.github/workflows/docker_build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build Docker image
+
+on: push
+
+env:
+  IMAGE_NAME_LIQUIDE_BLEIBEN: liquide-bleiben
+  IMAGE_NAME_CHATBOT: chatbot
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [$IMAGE_NAME_LIQUIDE_BLEIBEN, $IMAGE_NAME_CHATBOT]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build the Docker image $IMAGE_NAME_LIQUIDE_BLEIBEN
+      run: docker build . --file Dockerfile --tag $IMAGE_NAME_LIQUIDE_BLEIBEN:$(date +%s)
```

Comment:
I think this build will not work, it is missing a build arg (CB_AUTH).
Let's add this to the github secrets and include it in the build command

Reply:
Build worked in my forked Github repository. I would recommend **not** to use the Codebeam credentials for the docker build. Then the credentials will be published with the Docker image. As discussed with @Delet0r I recommend to set the environment variable when starting the Docker container `docker run -e ...`

Reply:
Happy to do that, but the Dockerfile needs to be setup correctly then

Reply:
What are you missing inside the `Dockerfile`? :)